### DISCUSSION TITLE
dropdown menu component by default removes the browsers scrollbar and prevents users from navigating

### DISCUSSION THREAD

**Discussion by MonkeeMan1** [shadcn-ui/ui#4552] [General]
Hello, I'm using the shadcn dropdown menu component and there is some default styles which I do not like. Whenever you open the dropdown it stops the user from being able to scroll, and also removes the browsers scroll bar. How do i disable this?
https://ui.shadcn.com/docs/components/dropdown-menu

I have gone through the component and removed all "overflow-hidden" styles, but still have the problem. I'm not sure what style is causing this issue, or how to fix it.

**Comment by Ishaan2053** [comment]
This is what worked for me. Add this to the global CSS file: -

```
html {
  @apply overflow-x-hidden;
}

body {
  @apply  !min-w-full;
}
```
